views/games.hbs

```diff
@@ -0,0 +1,5 @@
+<h2>{{username}} Games</h2>
+
+<a href="https://prpedrosa.github.io/Project_1_game/" target="_blank">Mr. Noobs</a>
+<hr>
+<a href="/profile">Back to profile</a>
```

Comment:
Since we changed the route, let's add the username here

`<a href="/profile/{{username}}">Back to profile</a>`